src/Symfony/Bundle/FrameworkBundle/Kernel/MicroKernelTrait.php

```diff
@@ -28,7 +28,7 @@
      * Add or import routes into your application.
      *
      *     $routes->import('config/routing.yml');
-     *     $routes->add('/admin', 'AppBundle:Admin:dashboard', 'admin_dashboard');
+     *     $routes->add('/admin', 'App\Controller\AdminController::dashboardAction', 'admin_dashboard');
```

Comment:
I suggest to drop  the `action` suffix (as recommended by https://symfony.com/doc/current/best_practices/controllers.html#controller-action-naming)